src/plugins/global-settings/typography/index.js

```diff
@@ -162,6 +162,7 @@ addFilter( 'stackable.global-settings.inspector', 'stackable/global-typography',
 					] }
 					value={ applySettingsTo }
 					onChange={ changeApplySettingsTo }
+					default={ 'blocks-stackable-native' }
```

Comment:
Because this is a static string, it's better if you pass it as a string instead of using an expression. This wasn't in the linter before, I just added it in `develop` now
```suggestion
					default="blocks-stackable-native"
```